ompi/mpi/man/man3/MPI_File_create_errhandler.md

```diff
@@ -0,0 +1,77 @@
+# Name
+
+MPI_File_create_errhandler  - Creates an MPI-style error handler
+that can be attached to a file.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_File_create_errhandler(MPI_File_errhandler_function *function,
+
+    MPI_Errhandler *errhandler)
+```
+
+## Fortran Syntax
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_FILE_CREATE_ERRHANDLER(FUNCTION, ERRHANDLER, IERROR)
+    EXTERNAL    FUNCTION
+    INTEGER    ERRHANDLER, IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_File_create_errhandler(file_errhandler_fn, errhandler, ierror)
+    PROCEDURE(MPI_File_errhandler_function) :: file_errhandler_fn
+    TYPE(MPI_Errhandler), INTENT(OUT) :: errhandler
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+
+# Deprecated Type Name Note
+
+MPI-2.2 deprecated the `MPI_File_errhandler_fn` and
+`MPI::file::Errhandler_fn` types in favor of `MPI_File_errhandler_function`
+and `MPI::File::Errhandler_function`, respectively. Open MPI supports both
+names (indeed, the `_fn` names are typedefs to the `_function` names).
```

Comment:
```suggestion
MPI-2.2 deprecated the `MPI_File_errhandler_fn` type in favor of
`MPI_File_errhandler_function`. Open MPI supports both
names (indeed, the `_fn` names are typedefs to the `_function` names).
```

I manually removed the C++ names, which are long gone / deleted out of Open MPI.